README.md

```diff
@@ -1,59 +1,74 @@
-# \<link[rel="lazy-import"]\> [![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+# \<link rel="lazy-import"\> 
 
-Declarative lazy HTML imports
+[![Build Status](https://img.shields.io/travis/Polymer/lazy-imports.svg?style=flat-square)](https://travis-ci.org/Polymer/lazy-imports)
+
+This repo implements declarative, lazy, HTML Imports. Normal HTML Imports are eager, meaning that they are loaded and evaluated in order first, before any code that follows. There is also a [javascript API](https://www.html5rocks.com/en/tutorials/webcomponents/imports/) for performing HTML Imports, allowing lazy loading of code at any point in time. Lazy loading lets you build an app in a sharded way, so that only the minimal amount of code is loaded to display the current view. This is a key piece of [the PRPL pattern](https://developers.google.com/web/fundamentals/performance/prpl-pattern/).
+
+What this repo adds on top of the above is a declarative way to describe the resources that you will import lazily, and a small mixin for using those declarative imports. The [polymer analyzer](https://github.com/Polymer/polymer-analyzer), [polymer linter](https://github.com/Polymer/polymer-linter), and [polymer bundler](https://github.com/Polymer/polymer-bundler) all understand these declarative lazy imports, giving you accurate lint warnings and sharded bundling without any configuration needed.
+
+To use lazy imports, write an HTML import as usual, except:
+
+  1) give it a rel of `lazy-import` instead of `import`
+  2) give each import a `group` attribute, you'll use that as a key later
```

Comment:
Suggest: "give each import a `group`  attribute; you'll use this as a key later"